scripts/zones/Ordelles_Caves/mobs/Bombast.lua

```diff
@@ -8,33 +8,10 @@ local entity = {}
 
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.AUTO_SPIKES, 1)
-    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 45, 0, 0)
+    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 15, 0, 0)
```

Comment:
Please leave a TODO note here for us to revisit this after scripted spikes are repaired, as this will not give retail accurate spikes behavior as is.

Reply:
TODO comment added